ansible/playbooks/roles/repository/files/download-requirements/src/command/dnf.py

```diff
@@ -1,21 +1,29 @@
 from typing import Dict, List
-import re
 
 from src.command.command import Command
 from src.error import CriticalError
 
 
-def filter_non_critical_errors(stderr: str):
-    re.sub('^Failed to set locale, defaulting to .+\n', '', stderr)
-
-class Dnf(Command):
+class DnfBase(Command):
     """
-    Interface for `dnf`
+    Base class for `dnf` interfaces
     """
 
     def __init__(self, retries: int):
         super().__init__('dnf', retries)
 
+    def _filter_non_critical_errors(self, stderr: str) -> str:  # pylint: disable=no-self-use
+        output_lines = [line for line in stderr.split(
+            '\n') if not line.startswith('Failed to set locale, defaulting to')]
```

Comment:
```python
        output_lines = [line for line in stderr.split('\n')
                        if not line.startswith('Failed to set locale, defaulting to')]
```

Reply:
This block was auto-formatted by IDE with `autopep8`, however it's possible to "enforce" your version which is also my preference 😉